x-pack/plugins/lens/public/embeddable/embeddable_factory.ts

```diff
@@ -21,7 +21,7 @@ import { UiActionsStart } from '../../../../../src/plugins/ui_actions/public';
 import { Start as InspectorStart } from '../../../../../src/plugins/inspector/public';
 import { Document } from '../persistence/saved_object_store';
 import { LensAttributeService } from '../lens_attribute_service';
-import { DOC_TYPE } from '../../common';
+import { DOC_TYPE } from '../../common/constants';
```

Comment:
why this change?

Reply:
The main problem with` export *` is, It cause excessive exports in required bundle see #110891.  On the other hand if we are talking about build package it's important to don't load extra modules.  Yes, I agree that it probably doesn't look professional, but it allow us optimize bundle size a little bit.

![image](https://user-images.githubusercontent.com/20072247/132030804-dc38848d-24ee-4308-8485-d1114b30c252.png)

Reply:
I'm fine with having specific `export` directives, but in this case I'm unsure about the real gain here.

Do we have any measurable difference between `import { DOC_TYPE } from '../common'` and `import {DOC_TYPE} from '../common/constants'`? The item is exported via `common/index` which is pretty much imported fully imported in the setup phase of Lens.
Isn't treeshaking already taking care of all these esm exporting implementation details?

Reply:
maybe if we avoid of using `export *` it starts to work correctly, but I found extra code in final bundle from which we used only constants. Replacing to '../common/constants' solved that issue

Reply:
During the sync today Alexey demonstrated that changing this import gives us 4 KB of a smaller bundle. We agreed the change is worth it :)